Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -839,37 +840,38 @@
         "Type": "ViewportBehaviorPage",
         "Subcategory": "Systems",
         "About": "Behavior for listening element enter or exit viewport",
-        "CodeUrl": "https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI/Behaviors",
+        "CodeUrl": "https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Behaviors/Viewport",
         "CodeFile": "ViewportBehaviorCode.bind",
         "XamlCodeFile": "ViewportBehaviorXaml.bind",
         "Icon": "/SamplePages/ViewportBehavior/ViewportBehavior.png",
-        "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/animations/ViewportBehavior.md"
+        "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/helpers/ViewportBehavior.md"
       },
       {
         "Name": "AutoFocusBehavior",
         "Type": "AutoFocusBehaviorPage",
         "Subcategory": "Systems",
         "About": "Behavior to automatically set the focus on a control when it loads",
-        "CodeUrl": "https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI/Behaviors/",
+        "CodeUrl": "https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Behaviors/Focus/AutoFocusBehavior.cs",
         "XamlCodeFile": "AutoFocusBehaviorXaml.bind",
         "Icon": "/Assets/Helpers.png",
-        "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/behaviors/AutoFocusBehavior.md"
+        "DocumentationUrl": "Cannot find!"
       },
       {
         "Name": "FocusBehavior",
         "Type": "FocusBehaviorPage",
         "Subcategory": "Systems",
         "About": "Behavior to automatically set the focus on the first control which accepts it",
-        "CodeUrl": "https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI/Behaviors/",
+        "CodeUrl": "https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Behaviors/Focus/FocusBehavior.cs",
         "CodeFile": "FocusBehaviorXaml.bind",
         "Icon": "/Assets/Helpers.png",
-        "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/behaviors/FocusBehavior.md"
+        "DocumentationUrl": "Cannot find!"
```

Comment:
We don't have one it was missing from the PR step here: https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3462

Not sure if we want to throw one together or not?

Reply:
Created a doc here: https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/481